client/extensions/woocommerce/app/settings/email/email-settings/index.js

```diff
@@ -12,28 +13,209 @@ import { connect } from 'react-redux';
  */
 import {
 	fetchEmailSettings,
+	emailSettingChange,
+	emailSettingsSubmitSettings,
 } from 'woocommerce/state/sites/settings/email/actions';
+import {
+	getEmailSettings,
+	areEmailSettingsLoading,
+	areEmailSettingsLoaded,
+	emailSettingsSaveRequest,
+	isSavingEmailSettings,
+	emailSettingsSubmitSettingsError,
+} from 'woocommerce/state/sites/settings/email/selectors';
+import { errorNotice, successNotice } from 'state/notices/actions';
+import CustomerNotification from './components/customer-notification';
+import ExtendedHeader from 'woocommerce/components/extended-header';
+import InternalNotification from './components/internal-notification';
+import NotificationsOrigin from './components/notifications-origin';
+import List from 'woocommerce/components/list/list';
+import ListHeader from 'woocommerce/components/list/list-header';
+import ListItemField from 'woocommerce/components/list/list-item-field';
+
+const originNotifications = [
+	{
+		field: 'email',
+		option: 'woocommerce_email_from_name',
+		title: translate( 'From name' ),
+		subtitle: translate( 'Emails will appear in recipients inboxes \'from\' this name.' ),
+	},
+	{
+		field: 'email',
+		option: 'woocommerce_email_from_address',
+		title: translate( 'From address' ),
+		subtitle: translate( 'If recipients reply to store emails they will be sent to this address.' ),
+		checkEmail: true,
+	},
+];
+
+const internalNotifications = [
+	{
+		field: 'email_new_order',
+		title: translate( 'New order' ),
+		subtitle: translate( 'Sent when a new order is received.' ),
+	},
+	{
+		field: 'email_cancelled_order',
+		title: translate( 'Cancelled order' ),
+		subtitle: translate( 'Sent when a new order is marked \'cancelled\'.' ),
+	},
+	{
+		field: 'email_failed_order',
+		title: translate( 'Failed order' ),
+		subtitle: translate( 'Sent when a new order is marked \'failed\'.' ),
+	},
+];
+
+// id enabled
+const customerNotifications = [
+	{
+		field: 'email_customer_on_hold_order',
+		title: translate( 'Order pending payment' ),
+		subtitle: translate( 'Sent when an order is marked \'payment pending\'.' ),
+	},
+	{
+		field: 'email_customer_processing_order',
+		title: translate( 'Processing order' ),
+		subtitle: translate( 'Sent when an order is marked \'payment processing\'.' ),
+	},
+	{
+		field: 'email_customer_completed_order',
+		title: translate( 'Completed order' ),
+		subtitle: translate( 'Sent when an order is marked \'paid in full\'.' ),
+	},
+	{
+		field: 'email_customer_refunded_order',
+		title: translate( 'Refunded order' ),
+		subtitle: translate( 'Sent when an order is marked \'payment refunded\'.' ),
+	},
+	{
+		field: 'email_customer_new_account',
+		title: translate( 'New account' ),
```

Comment:
Hi! I've found a possible matching string that has already been translated **28** times:
`translate( 'Create a new account' )` *ES Score: 6*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).